cg/exc.py

```diff
@@ -290,5 +290,9 @@ class DeliveryMessageNotSupportedError(CgError):
     """Exception raised when trying to fetch delivery messages for unsupported workflows."""
 
 
+class DeliveryMessageNoDeliveryError(CgError):
```

Comment:
```suggestion
class DeliveryMessageNoneError(CgError):
```
Perhaps? Flavour

Reply:
Might be better haha I did not like my naming...

Reply:
Hm, why do we need an exception? I don't like using exceptions for non-exceptional behaviour

Reply:
Hm, might be nice to just raise an exception called `NoAnalysesToDeliver` or something

Reply:
Yeah, I was thinking that it might be nice to be able to see all errors for the Delivery Message flow by just searching for DeliveryMessage though... Might not be any value in doing so.

Reply:
And this is for when the data delivery option is "no-delivery"

Reply:
I will keep as is for now, and we can change it if it is annoying.